release.sh

```diff
@@ -101,8 +103,10 @@ sed -e "s/-0.0.0/-${RELEASE_VERSION}/g" "${ROOTDIR}/nexus-repositories.yaml" \
 mkdir "${BUILDDIR}/shasta-cfg"
 "${ROOTDIR}/vendor/github.com/Cray-HPE/shasta-cfg/package/make-dist.sh" "${BUILDDIR}/shasta-cfg"
 
-# Save cray/nexus-setup and quay.io/skopeo/stable images for use in install.sh
-vendor-install-deps "$(basename "$BUILDDIR")" "${BUILDDIR}/vendor"
+# Use a newer version of cfs-config-util that hasn't rolled out to other products yet
```

Comment:
Please move this definition into `common.sh`, together with other tools images definitions, and use

    artifactory.algol60.net/csm-docker/stable/cfs-config-util:5.0.0

instead of

    arti.hpc.amslabs.hpecorp.net/csm-docker-remote/stable/cfs-config-util:5.0.0

Network channel between our builders and arti.hpc is completely overloaded and can't provide sufficient bandwidth, we see delays and timeouts almost every time. Therefore we use copies of tools images from artifactory.algol60.net.